src/app.md

```diff
@@ -233,28 +233,35 @@ fn main() {
 <a name="ex-file-predicate"></a>
 ##  Recursively find all files with given predicate
 
-Use the file names and metadata to recursively find in the current directory JSON files modified within the last day.
-
 [![walkdir-badge]][walkdir] [![cat-filesystem-badge]][cat-filesystem]
 
+Use the file names and metadata to recursively find in the current directory 
+JSON files modified within the last day. Enabling follow_links ensures symbolic
+links are followed as if they were normal directories and files. If a symbolic 
+link is broken or is involved in a loop, an error is yielded.
+
 ```rust,no_run
-# #[macro_use]
-# extern crate error_chain;
+#[macro_use]
+extern crate error_chain;
 extern crate walkdir;
 
 use walkdir::WalkDir;
-#
-# error_chain! {
-#     foreign_links {
-#         WalkDir(walkdir::Error);
-#         Io(std::io::Error);
-#         SystemTime(std::time::SystemTimeError);
-#     }
-# }
+
```

Comment:
no reason to unhide the error boilerplate.
Also please do not modify the original code if there is no need